src/utils/fileReader/index.ts

```diff
@@ -21,7 +26,7 @@ const fileReaderFactory = (options: Partial<FileReaderOptions>) => {
     return file && config.maxFileSize && file.size > config.maxFileSize;
   }
 
-  function readFilePromise(file: File): Promise<FileReaderFile | Error> {
+  function readFile(file: File | Blob): Promise<FileReaderFile> {
```

Comment:
Renaming as after adding types to this document, including the return type in the function name is no longer helpful